app/views/projects/show.html.erb

```diff
@@ -14,6 +14,7 @@
         <thead class="table-header fixed-header">
           <tr class="project-table__row project-table__row--header">
             <th class="project-table__cell">Story Title</th>
+            <th class="project-table__cell">Status</th>
```

Comment:
we need to add one td for the totals at the end of the table to match the columns
![image](https://github.com/fastruby/points/assets/188464/b0905900-d011-4296-923e-4d303f0e659c)